waiter/integration/waiter/authentication_test.clj

```diff
@@ -121,6 +121,11 @@
                                                                  :headers {"content-type" "application/x-www-form-urlencoded"}
                                                                  :method :post)
             _ (assert-response-status response 303)
+            cookie-fn (fn [cookies name] (some #(when (= name (:name %)) %) cookies))
```

Comment:
Nitpick: This is a useful one for `client_tools`, maybe name it `extract-cookie`